### PR TITLE
Start creating EBook mode

### PR DESCRIPTION
**Pull request by finanalyst**
- the aim of this node is to create an epub version of the Raku Documentation suite
- An epub can be described as a web site encapsulated into a file.
- See issue #356 for initial design goals.

### CODE REVIEW

**Comment by coke** [comment]
Looks like this is just configs, approving.

**Review by coke — APPROVED** [review]

**Review by dontlaugh — APPROVED** [review]

**Comment by finanalyst** [comment]
@dontlaugh I am still not entirely clear how the docs-dev commit works? 
Am I correct in the following: If I commit to the branch, then the commit gets reflected in docs-dev.raku.org but not docs.raku.org ? 
If so, then the docs-dev branch should not be deleted after a merge. Correct?

**Comment by coke** [comment]
Unless you're deliberately doing the work in standalone chunks, I would not merge back to main frequently.

You can always recreate the branch if it gets pruned post merge if needed

Yes, committing to the branch should reflect on the dev site.

**Comment by dontlaugh** [comment]
GitHub deletes the branch after a merge. This is a good default 99% of the time.

Your changes will remain on docs-dev.raku.org, because the act of pushing to the "docs-dev" branch _triggers a build_ that creates a standalone artifact: a container with the tag "docs-dev".

This container will contain the assets generated from the commit that you pushed.

On the docs-dev.raku.org server, we configure our cron job to pull down the container artifact. The server does not use `git clone` for deployment. It doesn't use git at all.

To continue hacking, push up another branch named "docs-dev", and another build will get triggered, and the container artifact tagged "docs-dev" will be overwritten...and in 30 min the server will pull that most recent "docs-dev" tag.

Keep pushing commits to docs-dev and everything will be fine. Keep an eye on this webpage if you're interested in seeing the artifacts that get built/overwritten after each push

https://quay.io/repository/colemanx/raku-doc-website?tab=history

Quay.io is the container artifact registry. The buildkite CI system has push access to this registry.

**Comment by finanalyst** [comment]
@dontlaugh Thank you for the clarification - it helped me understand the work flow.
@coke I dont intend to push again until I have a working ebook. I am working on the main plugin - ebook-embed on my collection-plugin-development repo, and then I'll transfer to Raku/doc-website.

The main plan is to get a working epub with minimal CSS and then open up for review so we can get communal input on style / functionality etc.

**Pull request merged**
